data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,458 @@
+}
+
+obj_spidercloth = {
+   name = "obj_spidercloth_production",
+   title = _"Build a spider farm and a weaving mill",
+   number = 2,
+   body = objective_text(_"Spidercloth Production",
+      p(_[[We need some spidercloth urgently as we are running out of it. So it is essential to spare every piece we still have. Reduce the input of all buildings that consume spidercloth. Don’t forget the constructionsites.]]) ..
+      p(_[[After that we need to have a farm under all circumstances as it needs spidercloth to be build. Additionally we need a spiderfarm and a weaving mill. Later on a gold spinning mill would be helpful as well.]]))
+}
+
+obj_trading_post = {
+   name = "obj_trading_post",
+   title = _"Build a trading post",
+   number = 1,
+   body = objective_text(_"Upgrade our scout’s house to a trading post",
+   p(_[[To be able to pay the tributes to Kalitath, and keep contact with Jundlina. We need to enhance our scout’s house to a trading post.]]))
+}
+
+obj_tribute = {
+   name = "obj_tribute",
+   title = _"Pay all tributes to Kalitath",
+   number = 1,
+   body = objective_text(_"Send our trader to Kalitath to deliver the demanded goods",
+   p(_[[We need to send the wares to Kalitath with our trader. He will pick them up as soon as a batch is ready in his trading post.]]) ..
+   p(_[[The following batches need to be delivered:]]) ..
+   li(_[[Five batches with five logs.]]) ..
+   li(_[[Four batches with four coal.]]) ..
+   li(_[[Three batches with three iron.]]) ..
+   li(_[[Two batches with two gold.]]))
+}
+
+obj_tribute2 = {
+   name = "obj_tribute2",
+   title = _"Pay all tributes to Kalitath",
+   number = 1,
+   body = objective_text(_"Send our trader to Kalitath to deliver the demanded goods",
+   p(_[[We need to send the wares to Kalitath with our trader. He will pick them up as soon as a batch is ready in his trading post.]]) ..
+   p(_[[The following batches need to be delivered:]]) ..
+   li(_[[Ten batches with five logs.]]) ..
+   li(_[[Eight batches with four coal.]]) ..
+   li(_[[Six batches with three iron.]]) ..
+   li(_[[Four batches with two gold.]]))
+}
+
+obj_defeat_maletus = {
+   name = "obj_defeat_maletus",
+   title = _"Defeat Maletus",
+   number = 1,
+   body = objective_text(_"Defeat the empire troops led by maletus.",
+   p(_[[We need to finish the empire of to have a chance for making a new home.]]))
+}
+
+-- =======================================================================
+--                                  Texts
+-- =======================================================================
+
+defeated = {
+   {
+      title =_"Defeated!",
+      body = sidolus(_"Sidolus has bad news…",
+         -- TRANSLATORS: Sidolus
+         _([[Aaaargh, Jundlina. I don’t know how this could have happened, but Satul couldn’t protect us from the enemy. They have destroyed our last warehouse. So, we have lost this battle and our people is doomed!]])),
```

Comment:
```suggestion
         _([[Aaaargh, Jundlina. I don’t know how this could have happened, but Satul couldn’t protect us from the enemy. They have destroyed our last warehouse. Thus we have lost this battle and our people is doomed!]])),
```